composer.json

```diff
@@ -34,7 +34,8 @@
         "doctrine/dbal": "^2.5",
         "doctrine/migrations": "^1.0",
         "nicolopignatelli/valueobjects": "~3.0",
-        "sandrokeil/interop-config": "^0.3.1"
+        "sandrokeil/interop-config": "^0.3.1",
+        "prooph/event-store-doctrine-adapter": "^3.0"
```

Comment:
No adapter should be installed by default to force the user to choose one. Please remove